homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -172,18 +188,18 @@ def __init__(self, name, prior, observations, probability_threshold, device_clas
         ]
         self._probability_threshold = probability_threshold
         self._attr_device_class = device_class
-        self._attr_is_on = False
-        self._callbacks = []
+        self._attr_is_on = bool(False)
```

Comment:
if I do `self._attr_is_on = False` or `self._attr_is_on: bool = False` the mypy interpreter chooses `Literal[False]`

Reply:
`_attr_is_on` is defined on the base class, so just False is fine here.
```suggestion
        self._attr_is_on = False
```

Interpreters are also correct that inside `__init__`, it will only ever have `False` value, which is why the interpreter sees it as `Literal[False]`